demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -2417,6 +2463,15 @@ export class DemoMeetingApp
     );
   }
 
+  private areVideoFiltersSupported(): boolean {
+    if (this.defaultBrowserBehaviour.supportsCanvasCapturedStreamPlayback() && this.enableUnifiedPlanForChromiumBasedBrowsers)  {
```

Comment:
nit: you can just do return this.defaultBrowserBehaviour.supportsCanvasCapturedStreamPlayback() && this.enableUnifiedPlanForChromiumBasedBrowsers without the if-else

Reply:
Done